crates/builder-api/src/block_info.rs

```diff
@@ -13,8 +13,8 @@ pub struct AvailableBlockInfo<I: NodeType> {
     pub block_hash: BuilderCommitment,
     pub block_size: u64,
     pub offered_fee: u64,
-    pub signature: <<I as NodeType>::SignatureKey as SignatureKey>::PureAssembledSignatureType,
-    pub sender: <I as NodeType>::SignatureKey,
+    pub signature: <<I as NodeType>::BuilderSignatureKey as BuilderSignatureKey>::BuilderSignature,
```

Comment:
I don't think you need the fully qualified syntax for `NodeType` b/c it is a bound of `I`.